net/autossh/src/opnsense/mvc/app/models/VerbNetworks/Autossh/Autossh.xml

```diff
@@ -0,0 +1,388 @@
+                        <kex_algorithms_03>ecdh-sha2-nistp256</kex_algorithms_03>
+                        <kex_algorithms_04>ecdh-sha2-nistp384</kex_algorithms_04>
+                        <kex_algorithms_05>ecdh-sha2-nistp521</kex_algorithms_05>
+                        <kex_algorithms_06>diffie-hellman-group-exchange-sha256</kex_algorithms_06>
+                        <kex_algorithms_07>diffie-hellman-group16-sha512</kex_algorithms_07>
+                        <kex_algorithms_08>diffie-hellman-group18-sha512</kex_algorithms_08>
+                        <kex_algorithms_09>diffie-hellman-group-exchange-sha1</kex_algorithms_09>
+                        <kex_algorithms_10>diffie-hellman-group14-sha256</kex_algorithms_10>
+                        <kex_algorithms_11>diffie-hellman-group14-sha1</kex_algorithms_11>
+                    </optionvalues>
+                </kex_algorithms>
+
+                <local_forward type="TextField">
+                    <required>N</required>
+                    <mask>/^([0-9a-z\ \.\-\_\:\*]){0,256}$/i</mask>
+                    <validationmessage>Must provide valid SSH client LocalForward string</validationmessage>
+                </local_forward>
+
+                <log_level type="OptionField">
+                    <required>Y</required>
+                    <multiple>N</multiple>
+                    <default>INFO</default>
+                    <optionvalues>
+                        <QUIET>Quiet</QUIET>
+                        <FATAL>Fatal</FATAL>
+                        <ERROR>Error</ERROR>
+                        <INFO>Info</INFO>
+                        <VERBOSE>Verbose</VERBOSE>
+                        <DEBUG1>Debug Level 1</DEBUG1>
+                        <DEBUG2>Debug Level 2</DEBUG2>
+                        <DEBUG3>Debug Level 3</DEBUG3>
+                    </optionvalues>
+                </log_level>
+
+                <macs type="OptionField">
+                    <required>Y</required>
+                    <multiple>Y</multiple>
+                    <default>macs_01,macs_02,macs_03,macs_04,macs_05,macs_06,macs_07,macs_08,macs_09,macs_10</default>
+                    <optionvalues>
+                        <macs_01>umac-64-etm@openssh.com</macs_01>
+                        <macs_02>umac-128-etm@openssh.com</macs_02>
+                        <macs_03>hmac-sha2-256-etm@openssh.com</macs_03>
+                        <macs_04>hmac-sha2-512-etm@openssh.com</macs_04>
+                        <macs_05>hmac-sha1-etm@openssh.com</macs_05>
+                        <macs_06>umac-64@openssh.com</macs_06>
+                        <macs_07>umac-128@openssh.com</macs_07>
+                        <macs_08>hmac-sha2-256</macs_08>
+                        <macs_09>hmac-sha2-512</macs_09>
+                        <macs_10>hmac-sha1</macs_10>
+                    </optionvalues>
+                </macs>
+
+                <port type="IntegerField">
```

Comment:
You could also use PortField, no validation or min/max required

Reply:
Ahh cool - I did not know